src/lib/services/map/providers/google/markerHandle.ts

```
@@ -0,0 +1,57 @@
+import type {EventUnsubscribe, IMarkerHandle, LatLngLiteral, MarkerHandleOptions} from '$lib/interfaces/map';
```

ESLint

[error] Replace `EventUnsubscribe,·IMarkerHandle,·LatLngLiteral,·MarkerHandleOptions` with `⏎····EventUnsubscribe,⏎····IMarkerHandle,⏎····LatLngLiteral,⏎····MarkerHandleOptions,⏎`

(prettier/prettier)